api/overdrive.py

```diff
@@ -327,8 +327,10 @@ def get_fulfillment_link_from_download_link(self, patron, pin, download_link, fu
         # If this for Overdrive's streaming reader, and the link expires,
         # the patron can go back to the circulation manager fulfill url
         # again to get a new one.
-        if not fulfill_url:
+        if not fulfill_url and flask.request:
```

Comment:
This is an unrelated change to fix informational Overdrive scripts I sometimes write that run outside of a Flask environment.